Directory.Build.props

```diff
@@ -34,6 +34,16 @@
     <NetFrameworkCurrent>net472</NetFrameworkCurrent>
   </PropertyGroup>
 
+  <!-- Honor the generic RuntimeConfiguration property. -->
+  <PropertyGroup>
+    <RuntimeConfiguration Condition="'$(RuntimeConfiguration)' == ''">$(Configuration)</RuntimeConfiguration>
+    <RuntimeConfiguration Condition="'$(RuntimeConfiguration)' == ''">Debug</RuntimeConfiguration>
```

Comment:
RuntimeConfiguration will not be empty here because `$(Configuration)` is always passed as Debug by default if it is not specified.